provider/resources.go

```diff
@@ -3556,7 +3556,7 @@ func Provider() tfbridge.ProviderInfo {
 				ObjectTypeSpec: schema.ObjectTypeSpec{
 					Type: "string",
 				},
-				Enum: []*schema.EnumValueSpec{
```

Comment:
I'd like these changes on master please so it's easier to compile pulumi-aws against local pulumi :)